Finish very basic instruction for network mode installation (#61)
chainstack/graph-deployment: helmfile/README.md

## Prerequisites
You will need to install the following tools before proceeding
1. [kubectl](https://kubernetes.io/docs/tasks/tools/)
2. [helmfile](https://github.com/roboll/helmfile)
3. [helm-diff](https://github.com/databus23/helm-diff)

Make sure to setup the kubectl config to point to the right kubernetes instance, see `terraform/<respective cloud provider>` for instructions.

## Network mode
You can deploy graph node in network mode. In this case additional graph indexer components gets deployed, so that the node can join the graph network.

> This The Graph doc could be useful - https://thegraph.com/docs/indexing .

Reviewer: ```suggestion
More information about indexing could be found [here](https://thegraph.com/docs/indexing).
```

### Installation
Prepare values in `values` directory first. `<namespace>` can be any string of your choice.
```
helmfile -f helmfile-network.yaml -n <namespace> apply
```

### Install graph-cli and indexer-cli

> It's needed to configure indexer-agent rules and check status
> https://thegraph.com/docs/indexing#from-npm-packages

Reviewer: ```suggestion
Please, install graph-cli and indexer-cli according to the following [guide](https://thegraph.com/docs/indexing#from-npm-packages).
```

### Check indexer status
```
kubectl -n <namespace> port-forward svc/graphprotocol-indexer-agent 18000:8000
graph indexer status
```

### Interacting with indexer-agent

> First you need to run port-forward command:

Reviewer: ```suggestion
First, you need to run the port-forward command:
```

```
kubectl -n <namespace> port-forward svc/graphprotocol-indexer-agent 18000:8000
```

> Connect cli:

Reviewer: ```suggestion
Then, connect with graph-cli and check status:
```

```

> graph indexer connect http://localhost:18000
> ```
> 
> Request indexer status:
> ```
> graph indexer status

Reviewer: ```suggestion
graph indexer connect http://localhost:18000
graph indexer status
```

```

> Other useful commands for subgraphs and cost model configuration could be found there: https://thegraph.com/docs/indexing#usage-1

Reviewer: ```suggestion
Other useful commands for subgraphs and cost model configuration could be found [here](https://thegraph.com/docs/indexing#usage-1).
```

it suggests that there are configuration errors.

Some possible errors could be

> 1. Blank network providers found in `values/graphprotocol-node*` in `config.chains`, if the network wasn't configure don't remember to comment it out.

Reviewer: Tbh, didn't fully understand what you meant here. Maybe the following line describes what you want to say:

```suggestion
Don't forget to fill in the `config.chains` value in the `values/graphprotocol-node*` file.
```
+
maybe `Some possible errors could be` line can be removed.
